MREUnityRuntime/MREUnityRuntimeLib/Messaging/Protocols/Protocol.cs

```diff
@@ -53,16 +53,10 @@ public void Stop()
 			}
 		}
 
-		private void Conn_OnReceive(string json)
+		private void Conn_OnReceive(Message message)
 		{
 			try
 			{
-#if ANDROID_DEBUG
-				App.Logger.LogDebug($"Recv: {json}");
-#endif
-
```

Comment:
This is useful debug logging for when android breaks and shouldn't be just removed I don't think.

Reply:
Unintentional. Will restore.

---
In reply to: [421119263](https://github.com/microsoft/mixed-reality-extension-unity/pull/151#discussion_r421119263) [](ancestors = 421119263)

Reply:
Ok.  Thanks!

---
In reply to: [421241477](https://github.com/microsoft/mixed-reality-extension-unity/pull/151#discussion_r421241477) [](ancestors = 421241477,421119263)

Reply:
The issue is there's no json here to log. It's getting deserialized within WebSocket. I have commented-out code there to reinstate message logging once another change comes in (a globally accessable logger -- WebSocket doesn't have a reference to `App`).

Reply:
We should track this somewhere to ensure we don't forget about this logging.  It is one of the more important logging statements for debugging android issues.

---
In reply to: [428248232](https://github.com/microsoft/mixed-reality-extension-unity/pull/151#discussion_r428248232) [](ancestors = 428248232)